src/cache_refresh/cache_refresh.cc

```diff
@@ -78,6 +84,9 @@ int refreshpasswdcache() {
         syslog(LOG_ERR, "Failure getting users, quitting");
         count = 0;
```

Comment:
is the count variable now unused due to changes below?

Reply:
Yes it is now unused. I removed it.